src/include/network/lan/address.rb

```diff
@@ -1355,6 +1355,19 @@ def AddressDialog
         Ops.set(@settings, "VLAN_ID", Builtins.tointeger(LanItems.vlan_id))
       end
 
+      if LanItems.type == "br"
+        ports = LanItems.bridge_ports.split(" ").select { |p| old_bridge_config?(p) }
+
+        unless ports.empty?
+          message = format(_("The bridge ports listed below are configured in the old way.\n\n" \
+                            "Bridge ports: %s\n\n" \
+                            "Do you want to adapt them now?"), ports.join(", "))
+          if Popup.YesNoHeadline(Label.WarningMsg, message)
+            ports.each { |p| configure_as_bridge_port(p) }
```

Comment:
@mvidner 

It was a doubt I have  at the beginning of the PBI as it was also commented in the card. After discussing it with @mchf the conclusion was that automatic conversion does not means not ask the user about it.

We could do it automatically, in that case when? when the interface is edited and saved or also do it although the user press cancel in the edition. If should be converted so why not just have a method that checks current interface configuration when the client is called and adapt old config style?

We just decided to prompt the user just to verify he really wanted to adapt the current configuration and also being a litle conservative taking in account that if something change in current workflow the network is restarted and should not restarted if not (so maybe the user does not expect changes when `OK` pressed but then he realized that the network was restarted and some files changed magically.